source/js/navigation.js

```diff
@@ -24,7 +24,7 @@ $(document).ready(() => {
     }
 
     // We check here whether the url contains /school/ section
```

Comment:
hmm isn't this comment outdated?

Reply:
No why should it?

Reply:
now you are checking for both school AND summer sections, so the comment is outdated. please update.